libfwupdplugin/fu-quirks.c

```diff
@@ -11,6 +11,10 @@
 #include <gio/gio.h>
 #include <string.h>
 
+#ifdef HAVE_SQLITE
```

Comment:
I wonder if we should just require it..

Reply:
There _was_ something we cared about that didn't have it -- openBMC IIRC.